package.json

```diff
@@ -57,6 +57,8 @@
   "main": "app.js",
   "private": true,
   "scripts": {
+    "docker:start": "docker-compose -f ./docker/docker-compose.yml up",
+    "docker:stop": "docker-compose -f ./docker/docker-compose.yml down",
```

Comment:
We should update docs/setup-development.md to describe the use of docker. In particular, I think that we'll want to  to describe:
1. The fact that docker is optional for local development
2. That you need to install [docker compose](https://docs.docker.com/desktop/install/mac-install/)
3. The start and stop commands
4. How to open the app in your browser once the container is up (I can't currently figure out how to do this 😅

Reply:
Also when do you need to rebuild the container? It looks like frontend changes hot reload, but backend changes won't (similar to local development now), so if you are working in docker and make a backend change you'll need to shut down the server with a keyboard interrupt or the stop command and then rerun `npm run docker:start`

Reply:
There's still an issue with this. I can't seem to get the client to talk to the server despite having the right ports forwarded, etc.

Reply:
This has been resolved as of the latest changes.